src/DataFrames.jl

```diff
@@ -16,6 +16,8 @@ using Reexport
 using GZip
 using SortingAlgorithms
 
+using FileIO  # remove after read_rda deprecation period
```

Comment:
Aha! I knew there was a reason I changed it. Thanks, @alyst. Perhaps you might want to update your PR, @andreasnoack?